pkg/cmd/controller/csr.go

```diff
@@ -0,0 +1,44 @@
+package controller
+
+import (
+	"context"
+	"github.com/openshift/cluster-policy-controller/pkg/operator"
+	"github.com/openshift/library-go/pkg/operator/csr"
+	"github.com/openshift/library-go/pkg/operator/events"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+const (
+	controllerName = "csr-approver-controller"
+	// Hardcoded for the Monitoring Operator:
+	// https://github.com/openshift/cluster-monitoring-operator/blob/master/manifests/0000_50_cluster-monitoring-operator_03-role-binding.yaml#L25-L27
+	monitoringServiceAccountNamespace = "openshift-monitoring"
+	monitoringServiceAccountName = "cluster-monitoring-operator"
+	monitoringCertificateSubject = "/CN=monitoring"
+)
+
+func RunCSRApproverController(ctx *ControllerContext) (bool, error) {
+	kubeClient, err := ctx.ClientBuilder.Client(infraClusterCSRApproverControllerServiceAccountName)
+	if err != nil {
+		return true, err
+	}
+
+	operatorClient := &operator.OperatorClient{
+	}
+
+	eventRecorder := events.NewRecorder(kubeClient.CoreV1().Events("test"), infraClusterCSRApproverControllerServiceAccountName, &v1.ObjectReference{})
+
+	controller := csr.NewCSRApproverController(
+		controllerName,
+		operatorClient,
+		kubeClient.CertificatesV1().CertificateSigningRequests(),
+		ctx.KubernetesInformers.Certificates().V1().CertificateSigningRequests(),
+		csr.NewLabelFilter(labels.Everything()),
```

Comment:
I hope this is a subject to change :slightly_smiling_face:

Reply:
what do we want to set here? :thinking:

Reply:
Exactly! So what shall we put here? @stlaz?

Reply:
configure a specific name/label for the CSR

Reply:
Oh, so an opinioated one for the sake of this use case? of course, that makes sense :+1: 
This would prevent auto-approving other CSRs from the same serviceaccont.
How about `openshift.io/metrics`?

Reply:
Ok, updated. Please take another look at it.